recipes/uwebsockets/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,7 @@
+#include "libusockets.h"
+#include "App.h"
```

Comment:
terrible, but it's the upstream idea ...

Reply:
uwebsockets examples usually require this include but ours don't. Dropped the include.